doc/README.md

```diff
@@ -1,7 +1,29 @@
 # Installation for OSX Development
-- Install gcc on your Mac via running ```gcc``` on the terminal
-- This will prompt you to install XCode
+TODO(raj): Write a script to do all this

```

Comment:
Generally, I think the script should just encapsulate dockerizing the environment vs more permanent configurations.